rtl/eclrtl/rtlfield.hpp

```diff
@@ -220,7 +222,7 @@ struct ECLRTL_API RtlKeyedIntTypeInfo final : public RtlTypeInfoBase
     inline __uint64 getUInt(const void * ptr) const { return (__uint64) getInt(ptr); }
     static unsigned __int64 addBias(__int64 value, unsigned length);
     static __int64 removeBias(unsigned __int64 value, unsigned length);
-    const RtlTypeInfo *child = nullptr;
+    // const RtlTypeInfo *child = nullptr;
```

Comment:
Why is this removed?

Reply:
Because it's unused.

I commented it out rather than delete it as I was unsure whether it SHOULD be used, though I suspect it's not needed.

Reply:
I suspect queryChildType() should be implemented for that type.  I would have thought it was useful for walking type information.